testing/src/test/java/io/stargate/it/http/graphfirst/FilesResourceTest.java

```diff
@@ -17,15 +17,33 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
 import io.stargate.it.http.graphql.GraphqlITBase;
 import java.io.IOException;
+import java.util.Map;
 import javax.ws.rs.core.MediaType;
 import okhttp3.OkHttpClient;
 import okhttp3.Request;
 import okhttp3.Response;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+@ExtendWith(CqlSessionExtension.class)
 public class FilesResourceTest extends GraphqlITBase {
```

Comment:
Nit: could you name the parent class `GraphqlTestBase` instead?
The existing integration tests all use "Test", I think it's better to stay consistent.